pulp-glue/pulp_glue/common/context.py

```diff
@@ -188,8 +207,12 @@ def call(
         Returns the operation result, or the finished task.
         If non_blocking, returns unfinished tasks.
         """
-        if parameters is not None:
-            parameters = preprocess_payload(parameters)
+        if parameters is None:
+            parameters = {}
+        if self.domain_enabled:
+            if "pulp_domain" in self.api.param_spec(operation_id, "path", required=True):
+                parameters["pulp_domain"] = self.pulp_domain
```

Comment:
This is correct, detail endpoints don't require the pulp_domain path parameter (since they already have it filled in) so we can't blindly add it to each call.